sql/core/src/test/resources/sql-tests/inputs/describe.sql

```diff
@@ -9,7 +9,7 @@ CREATE TEMPORARY VIEW temp_v AS SELECT * FROM t;
 CREATE TEMPORARY VIEW temp_Data_Source_View
   USING org.apache.spark.sql.sources.DDLScanSource
   OPTIONS (
-    From '1',
+    'From' '1',
```

Comment:
This looks like a breaking parser change which we should not do, @stefankandic .

Reply:
i see, any idea how to bypass this without breaking the parser? Not sure how we can keep FROM as a non-reserved keyword